### PR TITLE
Remove charge resolution tools, fixes #1634

### LINKED ISSUE #1634: remove charge resolution tools
We have a few tools like 
* ctapipe-chargeres-extract
* ctapipe-chargeres-hist
* ctapipe-chargeres-plot

Which probably are not needed anymore, as with the DL1 format and benchmarks, you can plot charge resolution pretty easily.   Have to check that ChargeResolutionCalculator does the same as what are in the protopipe benchmarks, but I think it's covered. @HealthyPear and @watsonjj should give the final say though.

### LINKED ISSUE #1643: Missing f-string in dl1_merge prevents from seeing broken filename

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +23,7 @@
     "ctapipe-stage1 = ctapipe.tools.stage1:main",
     "ctapipe-merge = ctapipe.tools.dl1_merge:main",
 ]
-tests_require = ["pytest"]
+tests_require = ["pytest", "pandas>=0.24.0"]
```
Comment (kosack) [inline setup.py:26]: isn't pandas still used in some of the notebooks generated by the docs?

Comment (maxnoe) [reply setup.py:26]: ha, yes, probably, will also add to `docs`

Comment (kosack) [reply setup.py:26]: that's another issue of course, we could probably clean them up to only use astropy

Comment (maxnoe) [reply setup.py:26]: seems not to be the case, the docs build passes...

Comment (maxnoe) [reply setup.py:26]: or the `read_table` function, yes.

Comment (maxnoe) [reply setup.py:26]: the docs build installed `[all]` not just `[docs]`.

**Review by kosack — APPROVED** [review]

**Review by HealthyPear — APPROVED** [review]

**Pull request merged**